src/command_modules/azure-cli-role/tests/test_graph.py

```diff
@@ -91,6 +91,34 @@ def body(self):
         self.cmd('ad app list --identifier-uri {}'.format(app_id_uri), checks=NoneCheck())
 
 
+class CreateForRbacScenarioTest(ScenarioTest):
+
+    def test_revoke_sp_for_rbac(self):
+        if not self.in_recording:
+            return
+        name = 'http://azurecli-test-revoke'
+        self.cmd('ad sp create-for-rbac -n ' + name)
+
+        self.cmd('ad sp list --spn ' + name)
+
+        self.cmd('ad app list --identifier-uri ' + name)
+
+        result = self.cmd('role assignment list --assignee ' + name).get_output_in_json()
+        object_id = result[0]['properties']['principalId']
+
+        self.cmd('ad sp delete --id ' + name)
```

Comment:
Does this scenario delete the application or the SP?  It seems like we should have a test for each (to make sure deleting the app deletes the SP)